drivers/sensor/isl29125/Kconfig

```diff
@@ -0,0 +1,14 @@
+# ISL29125 Color Light to Digital Converter configuration options
+
+# Copyright (c) 2016 Intive
+# SPDX-License-Identifier: Apache-2.0
+
+menuconfig ISL29125
```

Comment:
Just make the a config for now since there isn't anything below it. You can also remove the empty `if ISL29125..endif` block.